pylint_odoo/misc.py

```diff
@@ -112,14 +113,37 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
+        base_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
if you run manually the command `pylint` with ignore with other path won't work.
I mean, copy [test_repo](https://github.com/Vauxoo/pylint-odoo/tree/master/pylint_odoo/test_repo) to `/tmp/` and check if ignore parameters are working yet.